### ISSUE TITLE
malloc_consolidate(): invalid chunk size

### ISSUE THREAD

**Issue by morucci** [rpminspect/rpminspect#183]
Hello David,

Since 2020-07-31T17:00:49 all our rpminspect job runs fail. It seems there is a bug related to malloc_consolidate():
```
2020-08-04 12:42:56.643532 | TASK [rpminspect : Run rpminspect]
2020-08-04 12:44:18.972919 | container | malloc_consolidate(): invalid chunk size
2020-08-04 13:08:03.996949 | RUN END RESULT_TIMED_OUT: [untrusted : pagure.io/zuul-distro-jobs/playbooks/rpm/rpminspect.yaml@master]
2020-08-04 13:08:03.997120 | POST-RUN START: [untrusted : pagure.io/zuul-distro-jobs/playbooks/rpm/rpminspect-json2html.yaml@master]
```

- https://fedora.softwarefactory-project.io/zuul/build/adf1fe7eaabd4568a1a94052118a93eb
- https://fedora.softwarefactory-project.io/zuul/build/39497ac6d3e1412ca5c627ec72df63cc

Reported by @hroncok https://pagure.io/fedora-project-config/issue/104

Thanks in advance for your help

**Comment by dcantrell** [comment]
Thanks, working on this.  I noticed this problem locally yesterday too.

**Comment by dcantrell** [comment]
Fixed in commit cc7b3f2079438ce55a132f74d6b514e2a45cdcd2.
